pkg/restic/setup.go

```diff
@@ -121,8 +121,10 @@ func (w *ResticWrapper) setupEnv() error {
 
 	case storage.ProviderLocal:
 		r := w.config.Bucket
-		if err := os.MkdirAll(r, 0o755); err != nil {
-			return err
+		if _, err := os.Stat(r); os.IsNotExist(err) {
```

Comment:
This will not solve the CLI issue for UMD. We shouldn't create this directory from setup. Instead, we should create this directory from where we initialize repository.

Reply:
Do we actually need to create this directory during the initialization? Shouldn't the `restic init` command do this for us?

Reply:
Restic can't create the directory automatically for some cases. Specially for NFS, the directory must exist. That's why we needed to create the directory.